src/main/java/mekanism/common/tile/TileEntityFluidicPlenisher.java

```diff
@@ -337,44 +336,21 @@ public boolean hasTank(Object... data) {
     }
 
     @Override
-    public FluidStack drain(EnumFacing from, @Nullable FluidStack resource, boolean doDrain) {
-        if (resource != null && fluidTank.getFluid() != null && fluidTank.getFluid().getFluid() == resource.getFluid() && from == EnumFacing.UP) {
-            return drain(from, resource.amount, doDrain);
-        }
-        return null;
+    public int fill(EnumFacing from, @Nonnull FluidStack resource, boolean doFill) {
+        return fluidTank.fill(resource, true);
```

Comment:
Is the `true` there right? (I know its like that before your changes)

Reply:
I will look into it tomorrow if you don't end up doing so before then. I didn't even notice that it was specifically set to true, that is probably a good catch.